tests/testthat/test-api.R

```diff
@@ -0,0 +1,58 @@
+test_that("create_bugzilla_key works", {
+    vcr::use_cassette("create_bugzilla_key", {
+        cbk <- create_bugzilla_key(missing_host())
+        url <- "https://bugs.r-project.org/bugzilla/userprefs.cgi?tab=apikey"
+    })
+    expect_equal(paste0(missing_host(), "userprefs.cgi?tab=apikey"), url)
+    cli::test_that_cli("cli_alert_warning works", {
+        expect_snapshot({
+            cli::cli_alert_warning("This package hasn't been tested with this Bugzilla version.")
+        })
+    })
+})
+
+
+test_that("check_authentication works", {
+    vcr::use_cassette("check_authentication", {
+        ca <- check_authentication(missing_host())
+    })
+    expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
+    cli::test_that_cli("cli_alert_danger",{
+        expect_snapshot({
+            cli::cli_alert_danger("Not authenticated on this site.")
+            cli::cli_alert_success("Authenticated on this site!")
+        })
+    })
+})
+
+
+test_that("check_api_version works", {
+    vcr::use_cassette("check_api_version", {
+        cav <- check_api_version(missing_host())
+    })
+    expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
```

Comment:
Here `cav` is not used, so the function is never really tested. Please provide an expectation of what cav should be with `expect_equal(cav, ...)`